parsl/executors/high_throughput/executor.py

```diff
@@ -145,6 +145,11 @@
 
     encrypted : bool
         Flag to enable/disable encryption (CurveZMQ). Default is False.
+
+    manager_selector: ManagerSelector
+        Determines what strategy the interchange uses to select managers during task distribution.
+        See API reference under "Manager Selectors" regarding the various manager selectors.
+        Default: RandomManagerSelector()
```

Comment:
Do this and you'll get a hyperlink to the documentation for RandomManagerSelector:

```
$ git diff
diff --git a/parsl/executors/high_throughput/executor.py b/parsl/executors/high_throughput/executor.py
index 09e634f41d..92a256ae54 100644
--- a/parsl/executors/high_throughput/executor.py
+++ b/parsl/executors/high_throughput/executor.py
@@ -149,7 +149,7 @@ GENERAL_HTEX_PARAM_DOCS = """provider : :class:`~parsl.providers.base.ExecutionP
     manager_selector: ManagerSelector
         Determines what strategy the interchange uses to select managers during task distribution.
         See API reference under "Manager Selectors" regarding the various manager selectors.
-        Default: RandomManagerSelector()
+        Default: `RandomManagerSelector`
 """  # Documentation for params used by both HTEx and MPIEx
 
 
```